charts/etcd/templates/etcd-statefulset.yaml

```diff
@@ -99,8 +100,10 @@ spec:
 # {{- end }}
             - get
             - foo
-          initialDelaySeconds: 15
+            - --consistency="s"
+          initialDelaySeconds: 30
```

Comment:
I suggest to first stick to the `15s` as long as we don't see a need for it. Otherwise this setting will stretch the start up period for all etcds.